backend/src/Request/Admin/CaptainAmountFromOrderCash/CaptainAmountFromOrderCashFilterGetRequest.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace App\Request\Admin\CaptainAmountFromOrderCash;
+
+class CaptainAmountFromOrderCashFilterGetRequest
+{
+   private int $captainId;
+
+   private string $fromDate;
+   
+   private string $toDate;
```

Comment:
If the filtering options are not required, then we have to set also null as a var type.